### ISSUE TITLE
ArkInventory Search - Ghost Item at Top

### ISSUE THREAD

**Issue by Calesta** [arkayenro/arkinventory#782]
When I open the search window, the very first thing listed is black with a green square next to it.  A ghost item, if you will.  Is this something that can be fixed?

Screenshot:
http://imgur.com/toIGxxK

**Comment by Yanngo** [comment]
I've had the same issue (not a big problem) for some item. There was no tooltip. But today after some freezing (it takes a long time to search, really looking forward to fix to #708) I've linked the item in chat! What do you know? These are **Honor Points**. At least for me.

**Comment by arkayenro** [comment]
search is pretty instant for me, not much in the way of lag as i type in the search window (i usually have one the fires visible to check how bad the lag is).

i havent done anything to the code so im not sure why it doesnt appear to lag for me

**Comment by arkayenro** [comment]
btw, the list relies on the items being cached so the blank ones are ones you havent looked at for a while

**Comment by Yanngo** [comment]
The slow search looks like this: https://youtu.be/AnPTGkjV3b0
This time it was okay but the search for honor points was much slower.

**Comment by arkayenro** [comment]
thats really slow but it looks like you may have some out of date data in there (the green square is an item icon that no longer exists)

can you backup your saved variables and then run this `/ai cache erase confirm` which will erase all your saved data.  youll have to log back into each toon and access the bank/mail/void/vault but it hopefully will fix the issue

**Comment by arkayenro** [comment]
only 16 toons (only 11 at 100) across 2 profiles (although the search is global, it doesnt care about profiles), all with reagent banks, most with 30 slots bags, one private guild and the one i belong to (with the rest of my playable alts), then theres the mounts, pets, and toys :)

try 30700 beta 2, it might help.  you can also send me your saved variables file and i'll see what happens if i try the search on my machine

**Comment by arkayenro** [comment]
just want to double check, youve deleted your cache directory?

**Comment by Yanngo** [comment]
Yes, deleting cache is a matter of habit now. Since it is maintenance day, I've deleted the cache folder.

> try 30700 beta 2, it might help. you can also send me your saved variables file and i'll see what happens if i try the search on my machine

I deleted the saved settings file, installed the beta 2, and started setting things up but the window scale setting was not working. And Beta 1 was giving me huge FPS drops and freezes in Tempest Keep yesterday so I rolled back to 3.06.17 and started setting things up.

Starting from afresh will definitely work this time I am sure :) Except some addon still keeps resetting my toy filters to nothing. We'll live and see. Akuna matata and let's not worry about things much :)

**Comment by arkayenro** [comment]
my scale setting for the bank is working ok, but i did find an issue with the style/layout/catset not getting applied as soon as you change it (will be fixed in the next beta), it have been from that.

**Comment by Calesta** [comment]
My ghost item has corrected itself by disappearing so if you want to close this ticket you can unless you want to leave it open for the other issue around search that sort of hijacked it. :)

**Comment by Yanngo** [comment]
Ghost item has definitely disappeared after `/ai cache erase confirm`. And the search process is a bit faster now. So the solution is this macro and then relogging to every character.
